dotCMS/src/main/java/com/dotcms/rest/api/v1/template/TemplateHelper.java

```diff
@@ -66,7 +68,8 @@ public TemplateView toTemplateView(final Template template, final User user) {
                 .name(template.getName())
                 .friendlyName(template.getFriendlyName())
                 .title(template.getTitle())
-                .identifier(template.getIdentifier())
+                .identifier(FileAssetTemplateUtil.getInstance().isFileAssetTemplate(template)?
```

Comment:
I dont know if it is good idea do this here, maybe it is better in a TemplateAPI or IdentifierAPI, maybe a method getTemplateIdentifier